headers/modsecurity/transaction.h

```diff
@@ -323,7 +321,7 @@ class TransactionSecMarkerManagement {
     }
 
  private:
-    std::shared_ptr<std::string> m_marker;
+    std::shared_ptr<std::string> m_marker = nullptr;
```

Comment:
I think C++ gurantees this. `std::shared_ptr` is a class and therefore has a constructor and initialize itself.

Reply:
Yes. You're right. Fixed: https://github.com/owasp-modsecurity/ModSecurity/pull/3104/commits/cee3965f689ea52004d0e8b160054e227e6c5593